guides/hosting/performance/increment.md

```diff
@@ -0,0 +1,44 @@
+# Increment Storage
+
+{% hint style="info" %}
+This feature has been introduced with Shopware version 6.4.7.0
+{% endhint %}
+
+The increment storage is used in Shopware to show the current status of the message queue in the Administration or store the last used modules of the Administration users. This storage increments/decrements transaction safe the given key in the storage, leading to locks.
```

Comment:
```suggestion
The increment storage is used in Shopware to show the current status of the message queue in the Administration or store the last used modules of the Administration users. This storage increments/decrements transaction safe the given key in the storage, leading to locks.

The increment storage is used to store state and display it in the Administration. This can include

 * status of the message queue
 * last used module of administration users.
 
 This storage increments or decrements a given key in a transaction safe way, which causes locks upon the storage.
```